sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/ResourceManager.java

```diff
@@ -220,11 +222,10 @@ private ResourceManager(HttpPipeline httpPipeline, AzureProfile profile, SdkCont
                 .subscriptionId(profile.subscriptionId())
                 .buildClient();
 
-        // Unread in spot bugs
-//        this.subscriptionClientClient = new SubscriptionClientBuilder()
-//                .pipeline(restClient.getHttpPipeline())
-//                .host(restClient.getBaseUrl().toString())
-//                .buildClient();
```

Comment:
If it is not used, just delete the code.

Reply:
it is used in this code.

Reply:
I mean the codes that you've commented out.